napari/_qt/qt_event_loop.py

```diff
@@ -8,7 +8,10 @@
 from qtpy.QtWidgets import QApplication
 
 from napari import __version__
-from napari.utils.notifications import notification_manager
+from napari.utils.notifications import (
```

Comment:
Is there a reason this import is absolute? also the `from napari import __version__` is absolute too? maybe that was a mistake?

Reply:
fixed